File: src/test/java/test/CreateGroupTest.java
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class CreateGroupTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        CreateGroupReqDto request = new CreateGroupReqDto();
        request.setDescription("description_2223");
        request.setName("name_4433");
        request.setCode("code_926");

        GroupSingleRespDto response = client.createGroup(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/CreateGroupTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.CreateGroupReqDto{
        Description: "description_6051",
        Name: "name_5994",
        Code: "code_1238",
    }
    response := client.CreateGroup(&request)
    fmt.Println(response)
}

